.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,23 @@
+name: "Static Analysis"
+
+on:
+  pull_request:
+    branches:
+      - "master"
+    paths:
+      - .github/workflows/static-analysis.yml
+      - composer.*
+      - src/**
+      - phpstan*
+  push:
+    branches:
+      - "master"
+    paths:
+      - .github/workflows/static-analysis.yml
+      - composer.*
+      - src/**
+      - phpstan*
+
+jobs:
+  coding-standards:
+    uses: "doctrine/.github/.github/workflows/static-analysis.yml@4.0.0"
```

Comment:
That workflow assumes that you have PHPStan _and_ Psalm installed.

Reply:
Yes, I found out. Now I'm thinking about a) getting the essential stuff out of there b) doing the psalm config as well.